indico/modules/events/papers/blueprint.py

```diff
@@ -34,6 +34,12 @@
 
 # Management
 _bp.add_url_rule('/manage/papers/', 'management', management.RHPapersDashboard)
+_bp.add_url_rule('/manage/papers/deadlines/judging', 'manage_judging_deadline', management.RHSetJudgingDeadline,
```

Comment:
Or `<any(judge,content_reviewer,layout_reviewer):role>` if you reuse the enum (as I suggest in a comment below)